src/i106_data_stream.c

```diff
@@ -1274,6 +1280,7 @@ EnI106Status I106_CALL_DECL
         enReturnStatus = I106_WRITE_ERROR;
 #else
 // TODO - LINUX CODE
+    (void)suUdpHeaderNonF1Seg;

```

Comment:
Remove unused variable warning for variables that are used in windows but nor linux or vice versa